tests/Hangfire.PostgreSql.Tests/PostgreSqlJobQueueFacts.cs

```diff
@@ -481,14 +486,24 @@ public void Queues_Can_Dequeue_On_Notification()
 
         var task = Task.Run(() => {
           //dequeue the job asynchronously
-          job = queue.Dequeue(new[] { "default" }, new CancellationTokenSource(timeout).Token);
+          try
+          {
+            job = queue.Dequeue(new[] { "default" }, new CancellationTokenSource(timeout).Token);
```

Comment:
Cancellation token sources should be disposed once we're done with them. I see now that in the actual queue we also forgot to dispose the source, please do that too.